Project-3/Anshul275/README.md

```diff
@@ -0,0 +1,49 @@
+# Weather App
```

Comment:
Please add a link to your website and optionally screenshots of the functionality here itself.

Thanks!